CallsheetLocalizations/de.xcloc/Localized Contents/de.xliff

```diff
@@ -1556,12 +1556,13 @@
         <note>Header shown for people</note>
       </trans-unit>
       <trans-unit id="You must have a subscription to use the app." xml:space="preserve">
-        <source>You must have a subscription to use the app.</source>
+        <source>You must have a subscription to use the app.</source><target>Du brauchst ein Abo, um die App zu benutzen.</target>
         <note>Shown in Settings as a subtitle if you haven't purchased the app</note>
       </trans-unit>
       <trans-unit id="Your %@ will expire on&#10;%@" xml:space="preserve">
         <source>Your %1$@ will expire on
-%2$@</source>
+%2$@</source><target>Dein %1$@ läuft ab am
```

Comment:
@cliss Would need to see it in context with the variables resolved to be sure if the grammatical gender is correct here...

Reply:
Meaning, what values can `%1$@` take?